### PR TITLE
Fix copyright exclude filepath when running single maven module

### PR DESCRIPTION
**Pull request by mhdirkse**

### CODE REVIEW

**Comment by mhdirkse** [comment]
I cannot build the master anymore. This fixes the issue for me

**Comment by nielsm5** [comment]
License is already included in the file, perhaps you could remove the slash in front of the text instead.

**Review by nielsm5 — CHANGES_REQUESTED** [review]
License is already included in the file, perhaps you could remove the slash in front of the text instead.

**Review by nielsm5 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -706,5 +706,16 @@
 				</plugins>
 			</reporting>
 		</profile>
+		<profile>
+			<activation>
+				<file>
+					<exists>copyrightTemplateFile.txt</exists>
```
Comment (gvanbrakel) [inline pom.xml:712]: Toch een beetje raar om aan de hand van deze file te bepalen of het de root folder is.

**Review by gvanbrakel — APPROVED** [review]

**Pull request merged**